lib/screen/onboarding_page.dart

```diff
@@ -202,6 +209,35 @@ class _OnboardingPageState extends State<OnboardingPage>
     });
   }
 
+  Future<void> _restoreIfNeeded() async {
+    final configurationService = injector<ConfigurationService>();
+    if (configurationService.isDoneOnboarding()) return;
+
```

Comment:
I think we might need to add this. We are not sure 100% the keys from local had been restored, so I think we need to recall this. (That might be duplicated but it should be fine)
```
      await migrationUtil.migrationFromKeychain();
      await _accountService.androidRestoreKeys();
```